packages/suite-desktop-ui/src/support/DesktopUpdater/Downloading.tsx

```diff
@@ -84,6 +91,11 @@ export const Downloading = ({ hideWindow, progress }: DownloadingProps) => {
                     </>
                 )}
             </DownloadWrapper>
+            {!isCancelable && (
```

Comment:
Why shouldn't a user be able to download in the background outside of the settings page?

Reply:
It is always possible to download in the background. But while outside of settings page, it might not be clear that closing the modal means that the download continues in the background, so I replaced the default closing option with the button.

Reply:
I'd say it's not clear that it continues on background on any page :) Maybe it's worth always showing the `download in the bg` button?

Reply:
d52f5037b9181e4295bf4c7ccebaf07299681681
This modal now always look like this and is not closing by clicking outside:
![Screenshot 2023-03-16 at 14 29 39](https://user-images.githubusercontent.com/42465546/225653974-f762ad15-bf0e-4188-a1c2-9f500ee9aec3.png)